lua/nvim-tree/popup-menu/init.lua

```diff
@@ -0,0 +1,112 @@
+local a = vim.api
+
+local get_node = require('nvim-tree.lib').get_node_at_cursor
+local get_explorer = require('nvim-tree.core').get_explorer
+
+local M = {}
+
+M.is_open = false
+M.win = 0
+M.bufnr = 0
+
+local actions = {
+ ['copy'] = require('nvim-tree.actions.copy-paste').copy 
+}
+
+M.mappings = {
```

Comment:
We will need to move actions and mappings to a suitable place for user customisation, something like `view.popup.mappings/actions`, but that can come later.